### PR TITLE
Add User suspension

### PR DESCRIPTION
**Pull request by nicolas-brousse**
Add new feature to let admin be able to disable a user.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -12,16 +12,16 @@ def index
...
     end
   end
 
   def update
     if @user.update(secure_params)
-      redirect_to users_path, :notice => "User updated."
+      redirect_to users_path, notice: t(".flashes.updated")
     else
-      redirect_to users_path, :alert => "Unable to update user."
+      redirect_to users_path, alert: t(".flashes.can_be_updated")
```
Comment (B-Rass) [inline app/controllers/users_controller.rb:24]:
```suggestion
      redirect_to users_path, alert: t(".flashes.cant_be_updated")
```

File: config/locales/fr.yml [hunk 2]
```diff
@@ -550,7 +552,14 @@ fr:
         show_authentication_token: Afficher la clé API
         hide_authentication_token: Cacher la clé API
     flashes:
+      access_denied: Vous ne pouvez effectuer cette opréation
       authentication_token_reset: La clé API a été réinitialisée
+      created: Utilisateur créé
+      updated: Utilisateur mis à jour
+      can_be_updated: Une erreur est survenue pendant la mise à jour
```
Comment (B-Rass) [inline config/locales/fr.yml:559]:
```suggestion
      cant_be_updated: Une erreur est survenue pendant la mise à jour
```

File: spec/requests/users_request_spec.rb
Comment (B-Rass) [inline spec/requests/users_request_spec.rb:1]: Don't we want to add all missing tests here ?

**Review by B-Rass — CHANGES_REQUESTED** [review]

**Comment by B-Rass** [comment]
Review done @nicolas-brousse

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -43,7 +43,12 @@
 
         <% table.with_column do |user| %>
           <% unless user == current_user %>
-            <%= link_to t("action.delete"), user_path(user), data: { confirm: t("action.confirm") }, method: :delete, class: "button-xs btn-danger pull-right" %>
+            <% if user.suspended? %>
+              <%= link_to t("action.unsuspend"), unsuspend_user_path(user), data: { confirm: t("action.confirm") }, method: :patch, class: "button-xs btn-warning" %>
```
Comment (B-Rass) [inline app/views/users/index.html.erb:47]:
```suggestion
              <%= link_to t("action.unsuspend"), unsuspend_user_path(user), data: { confirm: t("action.confirm") }, method: :patch, class: "button-xs btn-succes" %>
```

Comment (B-Rass) [reply app/views/users/index.html.erb:47]: Better in green now ? or blue at least, but not the same color I think

**Review by B-Rass — CHANGES_REQUESTED** [review]

File: spec/requests/users_request_spec.rb
Comment (nicolas-brousse) [reply spec/requests/users_request_spec.rb:1]: Could I work on this in a separate pull request?

File: app/views/users/index.html.erb (hunk 3 above)
Comment (nicolas-brousse) [reply app/views/users/index.html.erb:47]: Ok for blue. I'll change it.

Comment (nicolas-brousse) [reply app/views/users/index.html.erb]: Changed

File: config/locales/fr.yml (hunk 2 above)
Comment (nicolas-brousse) [reply config/locales/fr.yml]: Fixed

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (nicolas-brousse) [reply app/controllers/users_controller.rb]: Fixed

File: spec/requests/users_request_spec.rb
Comment (B-Rass) [reply spec/requests/users_request_spec.rb:1]: Ok, here is the issue : https://github.com/nanego/my-dcim/issues/165

**Review by B-Rass — APPROVED** [review]

**Pull request merged**